package.json

```diff
@@ -6,10 +6,12 @@
   "license": "MIT",
   "scripts": {
     "dev": "NODE_ENV=development node server",
-    "build": "next build",
```

Comment:
Don't we want this as a part of the build process? Just curious

Reply:
It is, it's extended to include `build-storybook` too.

Reply:
was`build: next build && build-storybook` always there if so then I don't know what I was talking about.

Reply:
It was on the additions, but the deletions it had the line as deleted. That's potentially where the confusion came from? 😛